Cargo.toml

```diff
@@ -141,3 +141,6 @@ blocks_in_conditions = "allow"
 [[bin]]
 name = "spin"
 path = "src/bin/spin.rs"
+
+[patch.crates-io]
+wasmtime-wit-bindgen = { git = "https://github.com/calebschoepp/wasmtime", branch = "hotfix" }
```

Comment:
Waiting for this to be backported to Spin. The code can be reviewed in the meantime.